src/lib/classNames.ts

```diff
@@ -3,7 +3,7 @@ export default function classNames(
 ): string {
   const flatClasses = classes.map((className) => {
     if (typeof className === 'string') return className
-    if (Array.isArray(className)) return classNames(className)
+    if (Array.isArray(className)) return classNames(...(className as string[]))
```

Comment:
I think I missed when that function was initially introduced. Could you explain what we're using it for?

Reply:
It is a home made replacement of the [`classnames`](https://www.npmjs.com/package/classnames) library that those the same but this way we have one less dependency. It is useful to conditionally apply css classes based on booleans. Here the problem what that the classes were passed as on argument (an array) and now I am spreading them so they're passed as multiple argument.